community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/LabelIndexWriterStep.java

```diff
@@ -62,7 +60,7 @@ protected void process( Batch<InputNode,NodeRecord> batch, BatchSender sender )
     @Override
     public void close() throws Exception
     {
-        super.close();
         writer.close();
+        super.close();
```

Comment:
In a happy scenario (non-panic) this will not be a problem since `close()` will be called after all work has completed. I don't mind changing this order anyway.